pcsx2/CDVD/CdRom.cpp

```diff
@@ -748,11 +745,22 @@ void cdrWrite1(u8 rt)
 
 			cdr.Ctrl |= 0x80;
 			cdr.Stat = NoIntr;
+			cdr.SetlocPending = 1;
 			AddIrqQueue(cdr.Cmd, 0x800); // the seek delay occurs on the next read / seek command (CdlReadS, CdlSeekL, etc)
 		}
 		break;
-
+		do_CdlPlay:
```

Comment:
What is this? does pcsxr have the goto in the middle of a switch? how does that even work? O_o

Reply:
https://github.com/iCatButler/pcsxr/blob/62467b86871aee3d70c7445f3cb79f0858ec566e/libpcsxcore/cdrom.c#L653